vro-types/o11n-plugin-aria/index.d.ts

```diff
@@ -10,16 +10,429 @@ declare class VraHost {
+	): VraCloudAccountVsphere;
+
+	/**
+	 * Create a vSphere cloud account in an asynchronous manner
+	 * @param body
+	 */
+	createVSphereCloudAccountAsync(
+		body: VraCloudAccountVsphereSpecification
+	): VraRequestTracker;
+
+	/**
+	 * Update vSphere cloud account in an asynchronous manner
+	 * @param cloudAccountVsphere
+	 * @param body
+	 */
+	updateVSphereCloudAccountAsync(
+		cloudAccountVsphere: VraCloudAccountVsphere,
+		body: VraUpdateCloudAccountVsphereSpecification
+	): VraRequestTracker;
+
+	/**
+	 * Update vSphere cloud account
+	 * @param cloudAccountVsphere
+	 * @param body
+	 */
+	updateVSphereCloudAccount(
+		cloudAccountVsphere: VraCloudAccountVsphere,
+		body: VraUpdateCloudAccountVsphereSpecification
+	): VraCloudAccountVsphere;
+}
+
+declare class VraUpdateCloudAccountVsphereSpecification {
+	hostName: string;
+	acceptSelfSignedCertificate: boolean;
+	createDefaultZones: boolean;
+	password: string;
+	associatedCloudAccountIds: Object[];
+	regions: Object[];
+	dcid: string;
+	name: string;
+	description: string;
+	username: string;
+	tags: Object[];
+	constructor();
+	constructor();
+
+	/**
+	 * @param associatedCloudAccountIdsItem
+	 */
+	addAssociatedCloudAccountIdsItem(
+		associatedCloudAccountIdsItem: string
+	): VraUpdateCloudAccountVsphereSpecification;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraUpdateCloudAccountVsphereSpecification;
+
+	/**
+	 * @param regionsItem
+	 */
+	addRegionsItem(
+		regionsItem: VraRegionSpecification
+	): VraUpdateCloudAccountVsphereSpecification;
+}
+
+/**
+ * Specification for a vSphere cloud account.<br><br>A cloud account identifies a cloud account type and an account-specific deployment region or data center where the associated cloud account resources are hosted.
+ */
+declare class VraCloudAccountVsphereSpecification {
+	hostName: string;
+	acceptSelfSignedCertificate: boolean;
+	createDefaultZones: boolean;
+	password: string;
+	associatedCloudAccountIds: Object[];
+	regions: Object[];
+	dcid: string;
+	name: string;
+	description: string;
+	username: string;
+	tags: Object[];
+	constructor();
+	constructor();
+
+	/**
+	 * @param regionsItem
+	 */
+	addRegionsItem(
+		regionsItem: VraRegionSpecification
+	): VraCloudAccountVsphereSpecification;
+
+	/**
+	 * @param associatedCloudAccountIdsItem
+	 */
+	addAssociatedCloudAccountIdsItem(
+		associatedCloudAccountIdsItem: string
+	): VraCloudAccountVsphereSpecification;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraCloudAccountVsphereSpecification;
+}
+
+declare class VraRegionSpecification {
+	externalRegionId: string;
+	name: string;
+	constructor();
+	constructor();
+}
+
+declare class VraCloudAccountVsphere {
+	owner: string;
+	hostName: string;
+	linksExtension: string;
+	enabledRegionIdsExtension: string;
+	description: string;
+	orgId: string;
+	tags: Object[];
+	createdAt: string;
+	enabledRegions: Object[];
+	dcid: string;
+	readonly internalIdString: string;
```

Comment:
Is this intentionally left as the only one which is readonly? I am not familiar with the class itself, this is why i am asking

Reply:
Following the API this is an only one, which is `readonly`

![image](https://github.com/user-attachments/assets/83ac1c0e-2fe2-4dc8-8745-07c4ea9e285c)

Reply:
oooh good point, i entirely missed this column. thanks!